state/peerAccount.go

```diff
@@ -27,20 +26,15 @@ func NewEmptyPeerAccount() *peerAccount {
 }
 
 // NewPeerAccount creates new simple account wrapper for an PeerAccountContainer (that has just been initialized)
-func NewPeerAccount(address []byte, hasher hashing.Hasher, marshaller marshal.Marshalizer) (*peerAccount, error) {
+func NewPeerAccount(address []byte) (*peerAccount, error) {
 	if len(address) == 0 {
 		return nil, ErrNilAddress
 	}
 
-	tdt, err := NewTrackableDataTrie(address, nil, hasher, marshaller)
-	if err != nil {
-		return nil, err
-	}
-
 	return &peerAccount{
 		baseAccount: &baseAccount{
 			address:         address,
-			dataTrieTracker: tdt,
+			dataTrieTracker: disabled.NewDisabledTrackableDataTrie(),
```

Comment:
why do we use a disabled implementation now?

Reply:
The peer accounts do not have data tries, so there is no point in having an actual implementation of trackable data trie. There is a task to remove the `baseAccount` completely from the `peerAccount`. Will add a TODO.